app/modules/sightings/resources.py

```diff
@@ -376,13 +366,14 @@ def post(self, args):
                 'id': str(sighting.guid),
                 'version': sighting.version,
                 'encounters': result_data['encounters'],
+                'assets': assets,
             },
         }
         return rtn
 
 
 @api.route('/<uuid:sighting_guid>')
-@api.login_required(oauth_scopes=['sightings:read'])
+@api.login_required(oauth_scopes=[])
```

Comment:
huh?

Reply:
It's at the top of SightingByID, the thought was to ensure that anything below required a login, but to leave the scope to the individual ops if they choose to define it. If you really don't like it I can move it down, ha!

Reply:
It just seemed odd. But the upshot is now the get doesn't require 'sightings:read' which I think is a bug.
Not an expert but it seems strange to me to have the login required and the auth check at this level for any resources class as the permission is different for get/patch/post/delete. 
I think these should all be removed from the class level and moved to the method level.